kubernetes/zenko/values.yaml

```diff
@@ -117,17 +114,42 @@ zenko-queue:
     enabled: true
   configurationOverrides:
     "offsets.topic.replication.factor": 3 # - replication factor for the offsets topic
-    "auto.create.topics.enable": true     # - enable auto creation of topic on the server
+    "auto.create.topics.enable": false    # - enable auto creation of topic on the server
     "min.insync.replicas": 2              # - min number of replicas that must acknowledge a write
-  partitionCount: *nodeCount
-  partitionFactor: *crrParallelism
+    "message.max.bytes": "5000000"        # - the largest record batch size allowed
   prometheus:
     jmx:
       enabled: true
     kafka:
       enabled: true
-zookeeper:
-  enabled: false # Disables zenko-queue from deploying its own quorum
+  topics:
+    - name: backbeat-gc
+      partitions: 2
```

Comment:
What is the rationale for setting 2 as the default value? Why not `*nodeCount` as it used to be or `*replicaCount * *replicaFactor` to have one partition per running process, which seems a more reasonable default to me?

Reply:
Good catch, yes I meant *nodeCount.
That said, `*replicaCount * *replicaFactor` (or any sort of calculation that requires templating) will lead us into `yaml/helm-hell` and will require complicated, zenko-specific changes to the Zenko-Queue chart, so I don't consider it a possibilty.

Reply:
OK, still not great as the number of replicas should be *at least* the number of processes, but I get your concerns on manageability so we can start with this, but I think we will have to iterate again at one point.